src/tasks/daily/daily_battle_mixin.py

```
@@ -304,9 +310,35 @@ def battle(self):
+        nums_extra_run = max(0, int(self.config.get("体力刷完后继续刷取次数", 0) or 0))
+        if nums_extra_run > 0:
+            self.log_info(f"体力刷完后继续刷取次数: {nums_extra_run}")
+
         if left_ticket < stages_cost[category_name]:
-            self.log_info("体力不足")
-            return True
+            if nums_extra_run <= 0:
+                self.log_info("体力不足")
+                return True
+            else:
+                # 体力不足，执行额外刷取
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)